### ISSUE TITLE
Turning off "randomness seeded"

### ISSUE THREAD

**Issue by igtrnt** [catchorg/Catch2#2932]
My tests do not use any random generators bit I still get `Randomness seeded to: #######` message.
Is there a way to turn off rand-related stuff or silence this message?

**Comment by JoeyGrajciar** [comment]
I believe that should be in output only when you run tests in random order. For reference check https://github.com/catchorg/Catch2/blob/devel/docs%2Fcommand-line.md

**Comment by horenmar** [comment]
No, all built-in reporters report it. It guards both the output of random Generators, and shuffle order of tests, so it is important for reproducibility.


Currently the only way to disable the output is to provide your own reporter.
